core/src/main/scala/za/co/absa/spline/harvester/dispatcher/openlindispatcher/RESTResource.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.harvester.dispatcher.openlindispatcher
```

Comment:
```suggestion
package za.co.absa.spline.harvester.dispatcher.openlinage
```

Also, I think we have too many `dispatcher` words in the `za.co.absa.spline.harvester.dispatcher` folder - pretty much every class or package has it in its name. Filed a separate ticket for refactoring #415